Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -289,39 +289,39 @@ private void UploadDirectoryToGCSBucket(string directory, Dictionary<string, str
         {
```

Comment:
Line 284 should use GCS since it is an acronym.

Also, `relativeDirectoryName` sounds somewhat misleading. Would `gcsObjectNamePrefix` be more accurate?